gobblin-modules/gobblin-couchbase/scripts/install_test_deps.sh

```diff
@@ -10,5 +10,9 @@ if [ -d mock-couchbase/.git ];
     git -C mock-couchbase checkout b929a9f99a31a233c43bd94b6a696ad966206e02
   fi
 pushd mock-couchbase
-mvn package -Dmaven.test.skip=true
-popd
+if mvn package -Dmaven.test.skip=true; then
+  popd
+else
+  echo "Error building mock-couchbase"
```

Comment:
The output already had the error message:

./scripts/install_test_deps.sh: line 13: mvn: command not found

that is now followed up with the new message on the next line: Error building mock-couchbase

Since there was no check on the return value earlier, popd followed and execution continued without stopping.